docs/sources/reference/api/docker_remote_api_v1.19.md

```diff
@@ -333,6 +333,7 @@ Return low-level information on the container `id`
 			"NetworkDisabled": false,
 			"OnBuild": null,
 			"OpenStdin": false,
+            "PortSpecs": null,
```

Comment:
@runcom I see the other parts *do* use spaces, so I would also be okay with changing the tabs to spaces in this block